packer/README.md

```diff
@@ -0,0 +1,38 @@
+## PACKER
+
+This directory contains [packer](https://www.packer.io/) templates to create images from provisioned VMs
+
+### Local usage
+
+From this directory, use `packer build` followed by the json packer template to create an image
+
+For example: `packer build centos7-katello-devel-stable.json`
+
+You can create a package image using the above command and then add that box to vagrant:
+
+```
+vagrant box add --name centos7-katello-devel-stable centos7-katello-devel-stable.box
+```
+
+Then you can use it in a box definition. For the box built with `centos7-katello-devel-stable.json`, you will need to use the stable hostname.
+
+```
+centos7-katello-devel-stable:
+  box_name: centos7-katello-devel-stable
+  hostname: centos7-katello-devel-stable.example.com
```

Comment:
Specifying hostname is required given we auto-generate a hostname that would conflict?

Reply:
Can users set a custom hostname when deploying the stable box?

Reply:
Unfortunately no, since this is a box that has a katello install on it already, the hostname has to be fixed so the Katello install is under a hostname. The packer template overwrites the hostname with a `fix_hostname.sh` script so it is `centos7-katello-devel-stable.example.com` before katello is installed. 

Katello is then installed, the box is created and pushed to vagrant cloud. Then, when a dev spins up the box locally, they will need to create it with this hostname so it matches what Katello was installed with. This is why it is in the box definition

This does bring some limitations with multiple boxes. You could have multiple box definitions all using this hostname, but it will make DNS hard, you can only have one DNS entry for `centos7-katello-devel-stable.example.com` so that will have to be kept updated to the box you want it to point to. I believe you couldn't fudge the hostname DNS entry either because the cert won't match.